README.md

```diff
@@ -81,8 +81,8 @@ api.track_events(event)
 # Events tracked like this will be made part of a session; for independent events use :allow_offline
 api.track_events(event, allow_offline: true)
 
-# You can also track events and user attributes at the same time
-api.track_multi(event, attribute_hash)
+# You can also track events, user attributes, and device attributes at the same time. magic!
+api.track_multi(events: event, user_attributes: user_attributes, device_attributes: device_attributes)
```

Comment:
you also need to change this part of the README
https://github.com/lumoslabs/leanplum_api/blame/master/README.md#L91
to reflect the new method signature

Reply:
this still hasn't been addressed (otherwise looks great)

Reply:
done